src/mRides-server/Migrations/20170313042519_migration2.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace mRidesserver.Migrations
+{
+    public partial class migration2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Method names "Up" and "Down" might be a bit vague, maybe a more precise method name will follow the naming conventions we have used so far